crypto-common/src/main/kotlin/care/data4life/crypto/GCKeyPair.kt

```diff
@@ -15,8 +15,7 @@
  */
 
 package care.data4life.crypto
-
-import care.data4life.sdk.util.Serializable
+import care.data4life.crypto.util.Serializable
```

Comment:
why importing this if its not used? or i dont see it?

Reply:
Good question...will have a look, but I simply the old with new signature.

Reply:
I overlooked it myself - look what GCKeyPair is actually extending